docs/dictionary/constant/nothing.lcdoc

```diff
@@ -0,0 +1,19 @@
+Name: nothing
+
+Type: constant
+
+Syntax: nothing
+
+Summary:
+The value with no type
+
+Example:
+put nothing is strictly nothing -- true
+
+Description:
+The constant <nothing> is the only value with no type, thus
+it is the only value for which the <is strictly> <nothing>
+operator returns true.
+
+References:
+is strictly (operator)
```

Comment:
Is the operator not `is strictly nothing`? Or do we lump all the is striclty's together?

Reply:
We lump them all together. It might be better not to...

Reply:
That's probably something for when we refine the `is strictly type` operators and add `is strictly`.